### ISSUE TITLE
JSII Compile Error using Nutanix Provider

### ISSUE THREAD

**Issue by abstractalchemist** [hashicorp/terraform-cdk#2608]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### cdktf & Language Versions

cdtf version 0.15.3,0.15.0,0.14.3
go 1.18.3
node 18.14.0
WSL 2

### Affected Resource(s)

```cdktf init``` fails with jsii compile error.
### Debug Output

https://gist.github.com/abstractalchemist/534cd1e67741ab4c54b1795e70c9d524
### Expected Behavior

Generate nutanix local modules

### Actual Behavior

JSII compile error as indicated in the output
### Steps to Reproduce

Run ```cdk init --template=go --providers=nutanix/nutanix@1.7.1 --local```
### References

- #0000

**Comment by schapirama** [comment]
This is not just with this provider, but with any npm-based install of cdktf  (and not just 0.15.3, but I've tested almost everything from 0.13)

I think this is related to a breaking change in `@types/node` https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/64262 

The brew-based install of 0.15.3 works fine, as it seems to pre-package a version of `@types/node` from before the breaking change (the Feb 4th version, I think)

**Comment by abstractalchemist** [comment]
I managed a "workaround" by running
```
npm dist-tag @types/node
```

found that version 17.041 works with the version of typescript being used by JSII.  I installed it via
```
npm -g i @types/node@17.0.41
```

then went into the node_modules folder of cdktf-cli and deleted the node folder
```
cd /usr/local/node/lib/node_modules/cdktf-cli/node_modules/@types
rm -rf node
```

and created a soft link to the globally installed @type/node module
```
ln -s /usr/local/node/lib/@types/node /usr/local/node/lib/node_modules/cdktf-cli/node_modules/@types
```

This "fixed" the issue, but certainly isn't something that is long term.
```

**Comment by ansgarm** [comment]
Hi @abstractalchemist 👋

This should be fixed via v0.15.4, which we released yesterday! And thank you for sharing the workaround 👍
